source/Irrlicht/CMakeLists.txt

```diff
@@ -130,12 +130,9 @@ set(IRRLICHT_SRCS_COMMON
 	CColorConverter.cpp
 	CImage.cpp
 	${CMAKE_SOURCE_DIR}/src/irr/video/convertColor.cpp
-	${CMAKE_SOURCE_DIR}/src/irr/asset/CImageLoaderBMP.cpp
-	${CMAKE_SOURCE_DIR}/src/irr/asset/CImageLoaderDDS.cpp
```

Comment:
so you're nuking the DDS loader without providing a replacement?

Reply:
darn, I wiped it out thinking there wasn't any code using the DDS loader... apparently example 04 is still making some use of it.

putting the DDS one back in for the time being until a more robust solution appears (#253)